plugins/arDominionB5Plugin/js/advancedSearch.js

```diff
@@ -0,0 +1,209 @@
+(($) => {
+  "use strict";
+
+  $(() => $(".adv-search").each((_, el) => new AdvancedSearch(el)));
+
+  class AdvancedSearch {
+    constructor(element) {
+      this.$element = $(element);
+      this.$form = this.$element.find('form[name="advanced-search-form"]');
+      this.$reposFacet = this.$element
+        .find("#\\#facet-repository")
```

Comment:
Please, check this is working as noted in sync time.

Reply:
Fixed!